src/com/xilinx/rapidwright/design/DesignTools.java

```diff
@@ -2144,6 +2144,31 @@ public static Map<String,String> getInvertiblePinMap(Series series, Unisim unisi
 		return invertPinMap;
 	}
 
+	/**
+	 * Copy a cell from a design as a new design.
+	 * @param src The source design with or without implementation
+	 * @param cellName The full hierarchy cell name to be extracted as another design
+	 * @return A newly created design copied from the source
+	 */
+	public static Design createDesignFromCellWithStatic(Design src, String cellName) {
+		return createDesignFromCell(src, cellName, true);
+	}
+	public static Design createDesignFromCellWithoutStatic(Design src, String cellName) {
+		return createDesignFromCell(src, cellName, false);
+	}
+	public static Design createDesignFromCell(Design src, String cellName, boolean copyPIPOfStaticNet) {
```

Comment:
Instead of `copyPIPOfStaticNet`, how about `keepStaticRouting`?

Reply:
changed.